pkg/controller/kubelet-config/helpers.go

```diff
@@ -227,6 +262,15 @@ func getManagedFeaturesKeyDeprecated(pool *mcfgv1.MachineConfigPool) string {
 	return fmt.Sprintf("98-%s-%s-kubelet", pool.Name, pool.ObjectMeta.UID)
 }
 
+func getManagedNodeConfigKey(pool *mcfgv1.MachineConfigPool, client mcfgclientset.Interface) (string, error) {
+	return ctrlcommon.GetManagedKey(pool, client, "97", "kubelet", getManagedNodeConfigKeyDeprecated(pool))
+}
+
+// Deprecated: use getManagedNodeKey
+func getManagedNodeConfigKeyDeprecated(pool *mcfgv1.MachineConfigPool) string {
```

Comment:
I don't think you need to have this split out given there is no legacy case for node configs, but I think its not a big issue

Reply:
Addressed.